src/components/common/__tests__/Header.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { render, screen, fireEvent, waitFor } from 'utils/tests/testingRender';
+
+import routes from 'routes/routes';
+
+import Header from '../Header';
+
+describe('Header.tsx Tests', () => {
+  it('Should render header', () => {
+    render(<Header />);
+    const MenuIcon = screen.getByAltText('menu');
+    expect(MenuIcon).toBeInTheDocument();
+  });
+
+  it('Should show options when users do click and redirect to about', async () => {
+    render(<Header />);
+    const MenuIcon = screen.getByAltText('menu');
+    fireEvent.click(MenuIcon);
+
+    await waitFor(() => {
+      expect(screen.getByText('About')).toBeInTheDocument();
```

Comment:
since you are reusing these strings across the tests, wdyt about having them on a constant like:

```
const pages = {
  contact: "Contact", 
  about: "About",  
}
```

and then

```suggestion
      expect(screen.getByText(pages.about)).toBeInTheDocument();
```